### PR TITLE
Set Navigation Bar Text Color via SpannableString

### LINKED ISSUE #9824: [Bug] Android Shell FIRST ToobarItem grey-out when the Second ToobarItem CanExecute = False.
### Description
Android 8-10
Xamarin.Forms 4.5.0.356
Shell App
FIRST ToobarItem grey-out when the Second ToobarItem CanExecute = False.
### Steps to Reproduce
1.  Create Two ToobarItems or more.
2. Using MVVM pattern wire the Commands to them and also attach CanExecute.
I use Prism.Forms for this.  
3. When I click the last ToolbarItem, but the FIRST and LAST gets greys out together.

### Expected Behavior
FIRST ToolbarItem disable color should be INDEPENDENT of the other items.
### Actual Behavior
The FIRST button, what is not expected, gets greyed out and stays so, but still CLICKABLE.
### Basic Information

- Version with issue: 4.5.0.356
- Last known good version: Previous version had another issue with the CanExecute not working.
- IDE: Visual Studio 2019 16.4.5
- Platform Target Frameworks: 
  - Android: 10.1.40
- Nuget Packages:
    <PackageReference Include="Prism.Forms" Version="7.2.0.1422" />
    <PackageReference Include="Xamarin.Forms" Version="4.5.0.356" />
    <PackageReference Include="Xamarin.Essentials" Version="1.5.0" />

### Workaround
I add extra ToolbarItem button with empty text in front of others, then the isse goes unnoticed.

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change ###

When I rolled back setting the MenuId that broke the code that was used to retrieve the associated TextView so that only the first TextView ever got set to disabled looking.

- This generates a unique ViewId for each MenuItem
- This also sets the tint color via spannable string so I don't have to retrieve the TExtView anymore

### Issues Resolved ### 
- fixes #9824

### Platforms Affected ### 
- Android

### Testing Procedure ###
- platform ui tests provided
- click through control gallery and test various menu item permutations 

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ protected override void OnCreate(Bundle bundle)
 
 #if TEST_EXPERIMENTAL_RENDERERS
 			// Fake_Flag is here so we can test for flag initialization issues
-			Forms.SetFlags("Fake_Flag"/*, "CollectionView_Experimental", "Shell_Experimental"*/); 
+			Forms.SetFlags("UseLegacyRenderers"/*, "CollectionView_Experimental", "Shell_Experimental"*/);
```
Comment (PureWeen) [inline Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs:48]:
```suggestion
			Forms.SetFlags("Fake_Flag"/*, "CollectionView_Experimental", "Shell_Experimental"*/); 
```

File: Xamarin.Forms.Controls/GalleryPages/PlatformTestsGallery/PlatformTestsConsole.xaml.cs [hunk 2]
```diff
@@ -28,6 +29,7 @@ public partial class PlatformTestsConsole : ContentPage
 		int _testsRunCount = 0;
 
 		readonly PlatformTestRunner _runner = new PlatformTestRunner();
+		DisplaySettings _displaySettings;
```
Comment (PureWeen) [inline Xamarin.Forms.Controls/GalleryPages/PlatformTestsGallery/PlatformTestsConsole.xaml.cs:32]: These changes make it so if there is a test failure all the passed tests become non visible by default. This lets me see on the app center screen shot what is failing

File: Xamarin.Forms.Platform.Android.UnitTests/Issues.cs [hunk 3]
```diff
@@ -76,16 +77,16 @@ public async Task ToolbarItemWithReallyHighPriorityDoesntCrash()
 
 		[Test(Description = "No ObjectDisposedException should be thrown")]
 		[Issue(IssueTracker.Github, 5560, "[Android] Disposed EntryCell throws ObjectDisposed exception after updating an object that the EntryCell was previously bound to")]
-		public void EntryCellDisposed()
+		public async Task EntryCellDisposed()
```
Comment (PureWeen) [inline Xamarin.Forms.Platform.Android.UnitTests/Issues.cs:80]: When running locally this test kept locking up my device every 2 or three runs so I just adjusted it slightly.

I made sure to test that it's still accurately testing the intended use case.

File: Xamarin.Forms.Platform.Android.UnitTests/TestClasses/_5560Model.cs [hunk 4]
```diff
@@ -1,10 +1,21 @@
-﻿namespace Xamarin.Forms.Platform.Android.UnitTests
+﻿using System;
```
Comment (PureWeen) [inline Xamarin.Forms.Platform.Android.UnitTests/TestClasses/_5560Model.cs:1]: https://github.com/xamarin/Xamarin.Forms/pull/10404/files#r414204082

File: Xamarin.Forms.Platform.Android/Cells/EntryCellRenderer.cs [hunk 5]
```diff
@@ -40,7 +40,7 @@ public class EntryCellRenderer : CellRenderer
 
 		protected override void OnCellPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
-			if (_view == null || _view.IsDisposed())
+			if (!_view.IsAlive())
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.Android/Cells/EntryCellRenderer.cs:43]: what if `_view == null `  should we try `_view?.IsAlive`?

Comment (PureWeen) [reply Xamarin.Forms.Platform.Android/Cells/EntryCellRenderer.cs:43]: IsAlive is an extension method that returns false if _view is null

Comment (rmarinho) [reply Xamarin.Forms.Platform.Android/Cells/EntryCellRenderer.cs:43]: ah ok!

**Pull request merged**